packages/AnimationsEditor-Core.package/AnimationsEditorPropertyColor.class/instance/targetColorFromString..st

```diff
@@ -0,0 +1,5 @@
+accessing
+targetColorFromString: aColor 
+	self targetColor: (Color colorFrom: aColor asString). 
```

Comment:
method name and parameter suggests that a String is used instead of a Color
```suggestion
targetColorFromString: aString 
	self targetColor: (Color colorFrom: aString). 
```